sparse/impl/KokkosSparse_twostage_gauss_seidel_impl.hpp

```diff
@@ -985,8 +986,11 @@ class TwostageGaussSeidel {
                 Kokkos::subview(localZ, Kokkos::ALL(), range_type(j, j + 1));
             single_vector_view_t Rj(localRj.data(), num_rows);
             single_vector_view_t Zj(localZj.data(), num_rows);
+            my_exec_space
+                .fence();  // Wait for writes to R and Z to land, TODO: remove
             sptrsv_solve(handle->get_gs_sptrsvL_handle(), crsmatL.graph.row_map,
```

Comment:
My preference would be to call a `sptrsv_solve` overload that accepts an execution space instance. Alternatively, I could create std::vectors of size 1 and call `sptrsv_solve_streams`. What is your preference @brian-kelley, @lucbv, and @vqd8a?